### PR TITLE
Epmcuxda6632  delete observation

### PR DESCRIPTION
**Pull request by Leuvarden**

### CODE REVIEW

File: src/screens/DeleteObservation/DeleteObservation.js [hunk 1]
```diff
@@ -1,84 +1,101 @@
...
-  render() {
-    const { modalVisible } = this.state;
-    const onBackPress = () => {
-      this.setModalVisible(false);
-    };
-    const onDeletePress = () => {
-      this.setModalVisible(false);
-    };
+  const onDeletePress = () => {
+    console.log(navigation.state.params.ObservationItem.id);
```
Comment (AfanasyShevtsov) [inline src/screens/DeleteObservation/DeleteObservation.js]: Remove console.log please

File: src/screens/DeleteObservation/DeleteObservation.js [hunk 2]
```diff
@@ -1,84 +1,101 @@
...
-              label={translate("deleteObservation.deleteObservationInputLabel")}
-              textContentType="emailAddress"
-              containerStyle={styles.input}
+    service
+      .deleteObservation(
+        navigation.state.params.ObservationItem.id,
+        deleteDescription
+      )
+      .then(response => {
+        console.log(response);
```
Comment (AfanasyShevtsov) [inline src/screens/DeleteObservation/DeleteObservation.js]: Can we remove it?

File: src/screens/DeleteObservation/DeleteObservation.js [hunk 3]
```diff
@@ -1,84 +1,101 @@
...
+        setModalVisible(false);
+        navigation.navigate("Observations");
+      })
+      .catch(err => {
+        console.log(err);
```
Comment (AfanasyShevtsov) [inline src/screens/DeleteObservation/DeleteObservation.js]: We have to throw the error to show a message

**Review by AfanasyShevtsov — CHANGES_REQUESTED** [review]

File: src/screens/DeleteObservation/DeleteObservation.js (hunk 1 above)
Comment (Leuvarden) [reply src/screens/DeleteObservation/DeleteObservation.js]: fixed it

File: src/screens/DeleteObservation/DeleteObservation.js (hunk 2 above)
Comment (Leuvarden) [reply src/screens/DeleteObservation/DeleteObservation.js]: fixed it too

File: src/screens/DeleteObservation/DeleteObservation.js (hunk 3 above)
Comment (Leuvarden) [reply src/screens/DeleteObservation/DeleteObservation.js]: added alert to show message

**Review by AfanasyShevtsov — APPROVED** [review]

**Pull request merged**